Form-Controls/styles.css

```diff
@@ -0,0 +1,79 @@
+body {
+    font-family: Arial, sans-serif;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: 100vh;
+    margin: 0;
+    background-color: #f5f5f5;
+  }
+  
+  header, footer {
+    text-align: center;
+    margin: 20px 0;
+  }
+  
+  main {
+    flex-grow: 1;
+    display: flex;
+    align-items: center;
+  }
+  
+  form {
+    background: #ffffff;
+    padding: 20px;
+    border-radius: 8px;
+    box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+    width: 300px;
+  }
+  
+  h1, h2 {
+    font-size: 1.5em;
+    color: #333;
+    margin: 0;
+  }
+  
+  form h2 {
+    margin-bottom: 15px;
+    text-align: center;
+  }
+  
+  label {
+    font-weight: bold;
+    margin-top: 10px;
+    display: block;
+  }
+  
+  input[type="text"],
+  input[type="email"],
+  select,
+  input[type="date"],
+  button {
+    width: 100%;
+    padding: 8px;
+    margin-top: 5px;
+    border-radius: 4px;
+    border: 1px solid #ccc;
+    box-sizing: border-box;
+  }
+  
+  input:focus,
+  select:focus,
+  button:focus {
+    outline: none;
```

Comment:
You can style the outline directly. No need to remove the outline and add a border.  

Outline is useful because it does not take up any vertical space whereas border pushes the subsequent content down.